0000-report-inboxes.md

```diff
@@ -0,0 +1,72 @@
+
+# Motivation
+
+The current approach has some shortcomings:
+
+* Users are not currently able to bypass mods and report directly to admins - this may allow mods to conceal instance rule breaking in specific communities
+* Admins are not aware of community rules, so they may wish to take no action for most community rule breaking reports. However, if an admin resolves such a report, the relevant community mods most likely never see it.
+* Different instances may have different rules, but somebody resolving a report on one instance will resolve it for other instances as well, thus potentially resulting in missed reports.
+* Mods might take local action on a report and mark it as resolved even in cases where a user should be banned from the entire instance. In this case, admins are very unlikely to see the report.
+
+# Guide-level explanation
+
+### When creating reports, users will be able to select if it's a mod report, or an admin report (or both)
+
+![image](https://github.com/sunaurus/lemmy-rfcs/assets/5356547/9a21b527-6c88-4024-b287-3371d77688f4)
+
+### Instead of the current single report inbox, there will be three different kinds of inboxes
+
+* Admin reports - show all reports sent to admins (only visible to admins)
+* Mod reports - show all reports sent to mods for any communities the user moderates
+* All reports - Shows a read-only view of all (admin and mod) reports, only visible to admins
+   * This is a read-only version of the current 0.19.3 admin report view, and would allow admins to still keep an eye on mod actions on their instance if they wish
+    
+The UI wouldn't need to change for mods, but for admins, there would be a new selection at the top of the reports page (the "mod reports" tab would only be visible if the admin is also a mod in any community):
+![image](https://github.com/sunaurus/lemmy-rfcs/assets/5356547/cc4ad68c-6e85-4cd9-b324-131c06951cb3)
+
+### Resolving reports should be more granular
+
+* Reports in the "admin reports" tab can only be resolved for admins of the local instance
+   * To reduce overhead, **banning the reported user on the user's home instance + removing reported content should automatically resolve reports for remote admins as well**
+* Reports in the "mod reports" tab can only be resolved by relevant mods. Admins can only resolve these if they are also explicit mods in the relevant communities.
+   * To reduce overhead, **admins banning the reported user on the community instance OR the user's home instance + removing reported content should automatically resolve reports for mods as well**
+* Reports in the "all reports" tab can not be resolved, they are only there for a read-only overview
+   * This is to prevent cases of admins accidentally preventing mods from moderating according to their own community rules
```

Comment:
Admins *should* have the power of overruling mods though. If mods don't like the admin of the instance where their community lives, they should find a different instance to host their community.

Reply:
Sure, but I don't think that's relevant for resolving reports. I mean, admins can always remove content, ban users, etc. This point is more about preventing admins from inadvertently hiding reports from mods.